xapi_db_load/backends/clickhouse_lake.py

```diff
@@ -23,12 +22,18 @@ def __init__(
         db_username="default",
         db_password=None,
         db_name=None,
+        db_event_sink_name=None,
+        s3_key=None,
+        s3_secret=None,
     ):
         self.host = db_host
         self.port = db_port
         self.username = db_username
         self.database = db_name
+        self.event_sink_database = db_event_sink_name
         self.db_password = db_password
+        self.s3_key = s3_key
+        self.s3_secret = s3_secret
 
         self.event_raw_table_name = "xapi_events_all"
         self.event_table_name = "xapi_events_all_parsed"
```

Comment:
As in Aspects both of these databases can be configurable, shouldn't those be configurable too?